src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/xlf/ApplicationDesignerView.zh-Hans.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../ApplicationDesignerView.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANS" original="../ApplicationDesignerView.resx">
```

Comment:
@v-zbsail Would it be possible to update the tooling on your side to avoid changing this? It seems to change back and forth between "zh-Hans" and "zh-HANS" on every pull request, and adds a lot of noise.

Reply:
Language codes updated by post-process